src/logic/events/TrackpadEvents.swift

```diff
@@ -0,0 +1,150 @@
+import Cocoa
+
+fileprivate var eventTap: CFMachPort!
+fileprivate var shouldBeEnabled: Bool!
+
+//TODO: Should we add a sensetivity setting instead of these magic numbers?
+fileprivate let accVelXThreshold: Float = 0.05
+fileprivate let accVelYThreshold: Float = 0.075
+fileprivate var accVelX: Float = 0
+fileprivate var accVelY: Float = 0
```

Comment:
```suggestion
fileprivate let accVelXThreshold: Float = 0.03
fileprivate let accVelYThreshold: Float = 0.06
fileprivate var accVelX: Float = 0.35
fileprivate var accVelY: Float = 0.125
```

Thanks @lwouis for your work! I also tried your touch-pad and found the same problem with this PR. The sensitivity is not matched with Windows (in my experience). I tried different numbers and found that 0.03, 0.06, 0.35, and 0.125 are more sensitive and like how Windows works.